features/testbot/matching.feature

```diff
@@ -718,3 +718,27 @@ Feature: Basic Map Matching
           | abe   | 1,0.99973,1.00027,0.99973,1.00027,1        | depart,turn left,arrive  | Ok   |
           | ahd   | 1,0.99973,1.00027,0.99973,1.00027,0.999461 | depart,turn right,arrive | Ok   |
           | ahe   | 1,0.99973,1.00027,0.99973,1.00027,1        | depart,turn left,arrive  | Ok   |
+
+    @match @testbot
+    Scenario: Regression test - duration aggregation for phantom nodes
+        Given the profile "testbot"
+        Given a grid size of 10 meters
+        Given the node map
+          """
+          a--1-b2cd-3--e
```

Comment:
i  suggest  to make forward and reverse phantom node weights for `bc` segment different as  `a--1-b2-cd-3--e`